xbdd-web/src/models/Version.js

```diff
@@ -0,0 +1,34 @@
+class Version {
+  constructor(data) {
+    if (data) {
+      this.id = data["_id"];
+      this.major = data.coordinates.major;
+      this.minor = data.coordinates.minor;
+      this.servicePack = data.coordinates.servicePack;
+      this.pinnedBuildList = data.pinned ? data.pinned : [];
+      this.buildList = data.builds.reverse();
```

Comment:
Careful using reverse() because it actually modifies the input array. You can do .slice().reverse() to reverse without modifying it.